tests/validate_config_test.py

```diff
@@ -41,6 +41,20 @@ def conda_search_result():
 subdir      : linux-ppc64le
 url         : https://repo.anaconda.com/pkgs/main/linux-ppc64le/
 md5         : 4691146ff587f371f83f0e7bab93b63b
+dependencies: 
```

Comment:
Added a no timestamp version here too, though its not really a problem when doing env file validation at the moment.